golangci-lint gives errors for never used variables

- [x] I have looked at the documentation [here](https://pkg.go.dev/github.com/go-playground/validator/v10#section-documentation) first?
- [x] I have looked at the examples provided that may showcase my question [here](/_examples)?

### Package version eg. v9, v10: 



### Issue, Question or Enhancement:



### Code sample, to showcase or reproduce:

```
validator_test.go:2041:2: SA4006: this value of `current` is never used (staticcheck)
        current, _, _, ok = v.getStructFieldOKInternal(val, "Inner.CrazyNonExistantField")
        ^
validator_test.go:2044:2: SA4006: this value of `current` is never used (staticcheck)
        current, _, _, ok = v.getStructFieldOKInternal(val, "Inner.Slice[101]")
```

--- a/validator_test.go
+++ b/validator_test.go
@@ -2038,10 +2038,10 @@ func TestCrossNamespaceFieldValidation(t *testing.T) {
 	Equal(t, kind, reflect.String)
 	Equal(t, current.String(), "val2")
 
-	current, _, _, ok = v.getStructFieldOKInternal(val, "Inner.CrazyNonExistantField")
+	_, _, _, ok = v.getStructFieldOKInternal(val, "Inner.CrazyNonExistantField")
 	Equal(t, ok, false)
 
-	current, _, _, ok = v.getStructFieldOKInternal(val, "Inner.Slice[101]")
+	_, _, _, ok = v.getStructFieldOKInternal(val, "Inner.Slice[101]")
 	Equal(t, ok, false)
 
 	current, kind, _, ok = v.getStructFieldOKInternal(val, "Inner.Map[key3]")

